airflow-core/src/airflow/models/connection.py

```diff
@@ -478,7 +478,6 @@ def get_connection_from_secrets(cls, conn_id: str) -> Connection:
                 return conn
             except AirflowRuntimeError as e:
                 if e.error.error == ErrorType.CONNECTION_NOT_FOUND:
```

Comment:
This means it was not found at all. It checked all secrets backends and metadata DB and didn't find, so either removing this log or updating it to `log.debug("Unable to retrieve connection")` would make sense.

Reply:
Yea exactly, so I ended up removing that log to make it consistent with the non-task SDK behaviour. If we did want to include the log, maybe it should be something like "Connection was not found" to make it explicit that it didn't run into any errors and still couldn't find the connection

Reply:
It's alright this way.